discord/enums.py

```diff
@@ -728,6 +729,12 @@ class AppCommandPermissionType(Enum):
     channel = 3
 
 
+class AppCommandContext(Enum):
+    GUILD = 0
+    BOT_DM = 1  # this is DMs only
+    PRIVATE_CHANNEL = 2  # this is DMs + GDMs
```

Comment:
These should be lowercase as with other enums. Also, this class needs to be documented in api.rst manually.

```suggestion
    guild = 0
    bot_dm = 1  # this is DMs only
    private_channel = 2  # this is DMs + GDMs
```